src/components/TextInput/BaseTextInput/index.tsx

```diff
@@ -446,7 +447,7 @@ function BaseTextInput(
                     ]}
                     onLayout={(e) => {
                         let additionalWidth = 0;
-                        if (Browser.isMobileSafari() || Browser.isSafari()) {
+                        if (Browser.isMobileSafari() || Browser.isSafari() || Browser.isMobileChrome()) {
```

Comment:
I'm not sure how it will behave in other browsers, I think it will be safe to keep it limited to our supported browsers

Here are listed browser in `isMobile` function
`Android|webOS|iPhone|iPad|iPod|BlackBerry|BB|PlayBook|IEMobile|Windows Phone|Silk|Opera Mini`